test/index.js

```diff
@@ -89,3 +123,12 @@ describe('Klaviyo', function () {
     });
   });
 });
+
+/**
+ * Decode base64 and parse json
```

Comment:
Add JSDoc comments here; you should always document all functions with `@param`, `@return`, etc.—even if they're just test helpers